httpclient5/src/test/java/org/apache/hc/client5/http/impl/TestAuthSupport.java

```diff
@@ -0,0 +1,23 @@
+package org.apache.hc.client5.http.impl;
```

Comment:
@ITonyLi There are several code style violations in this test file. https://travis-ci.org/apache/httpcomponents-client/jobs/570372821 Could you please correct them? Otherwise the change-set looks good to me.

Reply:
Ok, I will change it